### PR TITLE
Changing Lightbend Cloudstate to Akka Serverless.

### PR DESCRIPTION
**Pull request by rstento**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,24 +63,24 @@ Build and push the container image to your container registry
 
 NOTE: This command builds and pushes the image directly to a container image repository bypassing local Docker (if it is present). However, it is possible to build the image using [Docker](https://www.docker.com/) with `./gradlew build jibDockerBuild`. Please refer to [Jib plugin documentation](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#build-to-docker-daemon) for further information.
 
-## Deploying to Lightbend Cloudstate
+## Deploying to Akka Serverless
 
-The following steps use `csctl` to deploy the application to [Lightbend Cloudstate](https://docs.lbcs.io/).
+The following steps use `csctl` to deploy the application to [Akka Serverless](https://docs.cloudstate.io/).
```
Comment (pvlugter) [inline README.md:68]: Assume that should be `docs.cloudstate.com`

```suggestion
The following steps use `csctl` to deploy the application to [Akka Serverless](https://docs.cloudstate.com/).
```

File: README.md [hunk 2]
```diff
@@ -63,24 +63,24 @@ Build and push the container image to your container registry
...
 
-If you're self-hosting Cloudstate, the instructions for deploying the sample shopping cart application are in the [`deploy` directory](./deploy/README.md)
+If you're writing Cloudstate services to deploy on your own Kubernetes cluster, the instructions for deploying the sample shopping cart application are in the [`deploy` directory](./deploy/README.md)
 
 ### Prerequisites
 
-* Get [Your Lightbend Cloudstate Account](https://docs.lbcs.io/gettingstarted/account.html)
-* Install [csctl](https://docs.lbcs.io/getting-started/set-up-development-env.html)
+* Get [Your Lightbend Cloudstate Account](https://docs.cloudstate.io/gettingstarted/account.html)
```
Comment (pvlugter) [inline README.md:74]: Even with `docs.cloudstate.com`, not sure this is the correct link (looks like old docs still available):

https://docs.cloudstate.com/gettingstarted/account.html

Suggesting this instead:

https://docs.cloudstate.com/getting-started/lightbend-account.html

```suggestion
* Get [Your Lightbend Cloudstate Account](https://docs.cloudstate.com/getting-started/lightbend-account.html)
```

File: README.md [hunk 3]
```diff
@@ -63,24 +63,24 @@ Build and push the container image to your container registry
...
+* Install [csctl](https://docs.cloudstate.io/getting-started/set-up-development-env.html)
```
Comment (pvlugter) [inline README.md:75]:
```suggestion
* Install [csctl](https://docs.cloudstate.com/getting-started/set-up-development-env.html)
```

**Review by pvlugter — COMMENTED** [review]
Links to `docs.cloudstate.io` should be `docs.cloudstate.com`

File: README.md (hunk 1 above)
Comment (rstento) [reply README.md:68]: Thanks for catching, I was obviously in too big of a hurry!

File: README.md (hunk 2 above)
Comment (rstento) [reply README.md:74]: It indeed changed to https://docs.cloudstate.com/getting-started/lightbend-account.html

File: README.md (hunk 3 above)
Comment (rstento) [reply README.md]: I think I got them all now.

**Review by pvlugter — APPROVED** [review]

**Pull request merged**